### PR TITLE
packaged as secure-handlebars

### PR DESCRIPTION
**Pull request by adon-at-work**
- branded as secure-handlebars
- facilitated client-side use by shipping distributable with handlebars, context-parser, and xss-filters bundled
- included integration test

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by adon-at-work** [comment]
@neraliu, please review. thanks.

File: Gruntfile.js [hunk 1]
```diff
@@ -16,13 +16,55 @@ module.exports = function(grunt) {
     },
     browserify: {
       standalone: {
-        src: [ 'src/<%= pkg.name %>.js' ],
+        src: 'src/<%= pkg.name %>.js',
         dest: 'dist/<%= pkg.name %>.js',
         options: {
           browserifyOptions: {
-            standalone: 'ContextParserHandlebars'
+            standalone: 'Handlebars'
```
Comment (neraliu) [inline Gruntfile.js]: it will pollute the name of original Handlebars, should we use "SecureHandlebars"?

**Comment by neraliu** [comment]
if we push this to live, what will happen to the exiting npm package of context-parser-handlebars? we should redirect the user to secure-handlebars?

File: Gruntfile.js (hunk 1 above)
Comment (adon-at-work) [inline Gruntfile.js]: browserify version is for client-side use like <script src="">
to meet the goal of changing only the URL, we should override the original Handlebars, right?

**Comment by adon-at-work** [comment]
this will become a new npm package called secure-handlebars. yes, if secure-handlebars is released, we can branch out from the latest version of context-parser-handlebars to write a deprecate message and ask the user to use the new one (much like from express3-handlebars to express-handlebars).

**Comment by neraliu** [comment]
agreed!

File: Gruntfile.js (hunk 1 above)
Comment (neraliu) [inline Gruntfile.js]: yes

**Pull request merged**